java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -914,6 +914,13 @@ private String getBlockName(Block b) {
         if (b != null) {
             String sName = b.getSystemName();
             String uName = b.getUserName();
+            // Jay Janzen
```

Comment:
Comments can (by design) be extensive and multiline, unlike the names, so I think the UI changes would be breaking for some users.